iceoryx_hoofs/testing/include/iceoryx_hoofs/testing/mocks/error_handler_mock.hpp

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2022 by Apex.AI Inc. All rights reserved.
```

Comment:
Why do we need it and cannot use the `setTemporaryErrorHandler`?

Please drop this change from the history as well.

Reply:
Now, the `ErrorHandler` is free from any test code, see more in my [previous comment](https://github.com/eclipse-iceoryx/iceoryx/pull/1100#discussion_r807093601).